src/app/app.module.ts

```diff
@@ -47,7 +47,6 @@ import {
 /*
  * Platform and Environment providers/directives/pipes
  */
-import { featureTogglesApiUrlProvider } from '../a-runtime-console/shared/feature-toggles.provider';
```

Comment:
Can the file `a-runtime-console/shared/feature-toggles.provider` be removed from the repository as well? Or is it still being used elsewhere?

Reply:
oops I miss to remove it, let me do it